app/uploaders/image_uploader.rb

```diff
@@ -6,25 +6,30 @@ class ImageUploader < WhitehallUploader
   end
 
   def extension_white_list
-    %w(jpg jpeg gif png)
+    %w(jpg jpeg gif png svg)
   end
 
-  version :s960 do
+  version :s960, if: :image? do
     process resize_to_fill: [960, 640]
   end
-  version :s712, from_version: :s960 do
+  version :s712, from_version: :s960, if: :image? do
     process resize_to_fill: [712, 480]
   end
-  version :s630, from_version: :s712 do
+  version :s630, from_version: :s712, if: :image? do
     process resize_to_fill: [630, 420]
   end
-  version :s465, from_version: :s630 do
+  version :s465, from_version: :s630, if: :image? do
     process resize_to_fill: [465, 310]
   end
-  version :s300, from_version: :s465 do
+  version :s300, from_version: :s465, if: :image? do
     process resize_to_fill: [300, 195]
   end
-  version :s216, from_version: :s300 do
+  version :s216, from_version: :s300, if: :image? do
     process resize_to_fill: [216, 140]
   end
+
+  def image?(new_file)
```

Comment:
Yes.. I think so. It looks like it gets called quite a bit throughout the tests with nil image. Not sure it would happen outside of testing but it seems like a safer bet to leave it in there.